tests/pipeline/test_python_component.py

```diff
@@ -85,4 +93,7 @@ def transform(self, dataframe: pd.DataFrame) -> pd.DataFrame:
         ref=AddN,
         produces={"x": pa.int32(), "y": pa.int32()},
         consumes={"x": pa.int32(), "y": pa.int32()},
+        arguments={"n": 1},
     )
+
+    DockerCompiler().compile(pipeline)
```

Comment:
This expands the coverage of this test by including the compilation without errors of lightweight components.